src/renderer.ts

```diff
@@ -201,7 +201,7 @@ export function patternFillArc(x: number, y: number, width: number, height: numb
     strt = 0;
     stp = Math.PI * 2;
   }
-  const increment = (stp - strt) / o.curveStepCount;
+  const increment = Math.min(Math.PI  / o.curveStepCount, (stp - strt) / 2);
   const points: Point[] = [];
```

Comment:
I'm not sure if this makes it better. Filling needs a higher resolution than stroking. In stroking you are connected sampled points with a curve. When filling, we are approximating it to a polygon so a greater number of points is actually useful. 
Perhaps I'm missing something. Can you share a sample where this makes it better? Thanks.